internal/utils/utils.go

```diff
@@ -0,0 +1,77 @@
+package utils
```

Comment:
I think we should have a local `go.mod` / `go.sum` for this particular tool, like we have for `tools`/`test` in `lorawan-stack`. Otherwise we will have import cycles when we try to use things like the schema and models.

Reply:
I countered the import cycles by using unnamed copy of the structure in the utils. Not quite sure how to set it up with a `go.mod` at that level. Couldn't get imports to work into the main module.

Reply:
I missed adding some context here. My proposal was to actually make the models part of a public (not `internal`) package that we can import into `lorawan-stack`, such that we avoid having the models in two places. If we want to do that, the package exposed by `lorawan-frequency-plans` should not import `lorawan-stack`, which why I proposed that we decouple the model from the docs and schema generation.

How this works in `lorawan-stack` is that the root contains the public `go.mod`/`go.sum` and the subfolder [contains](https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.24/tools/go.mod) it's own set of ` go.mod` / `go.sum`. Since the tools would import the top level model definitions, imports _should_ work. It is indeed the case that the reverse is not really possible (you cannot import the tool into the root project, but that's fine).